### PR TITLE
[Easy] Parametrize the polling frequency

### PR DESCRIPTION
**Pull request by fleupold**
Allow us to configure how often we want to poll without requiring any code change (so we can adapt it in kubernetes accordingly)

### Test Plan

CI and `yarn start` and see we are refetching every 10 seconds.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -23,6 +23,10 @@ const argv = yargs
     describe: 'The number of intermediate orderbooks to look at when computing the transitive one',
     default: 2,
   })
+  .option('pollFrequency', {
```
Comment (nlordell) [inline src/index.ts]: ubernit: slightly inconsistent arg names - would prefer is the casing (camelCase or snake_case) were consistent for all the args.

**Review by nlordell — APPROVED** [review]
:+1:

File: src/index.ts (hunk 1 above)
Comment (anxolin) [reply src/index.ts]: For cli arguments, usually kebap case is more common, right?

**Review by anxolin — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (nlordell) [reply src/index.ts]: From the yargs docs:
```
var argv = require('yargs')
  .env('MY_PROGRAM')
  .option('f', {
    alias: 'fruit-thing',
    default: 'apple'
  })
  .argv
console.log(argv)
```
```
$ node fruity.js
{ _: [],
  f: 'apple',
  'fruit-thing': 'apple',
  fruitThing: 'apple',
  '$0': 'fruity.js' }
```

So it looks like using kebab-case in the spec will give you camelCase properties. I wouldn't be surprised if using camelCase properties also allowed accepted arguments in kebab-case.

Comment (fleupold) [reply src/index.ts]: There is an [open issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/39255) that the typescript type derivation doesn't work for the camelCase properties. But we can access the kebap case ones via `object[string]` (cf. latest commit)

Comment (bh2smith) [reply src/index.ts]: Today I learned about kebab-case!

**Pull request merged**